### PR TITLE
Add UI customization section for Admin app

### PR DESCRIPTION
**Pull request by ruthryi**
A - 932

### CODE REVIEW

File: en/docs/develop/customizations/advanced-ui-customization.md [hunk 1]
```diff
@@ -78,6 +78,82 @@ The rest of the files are runtime configurations for eslint, jest, webpack, npm.
...
+
+1. Navigate to the `<API-M_HOME>/repository/deployment/server/jaggeryapps/admin` directory in a terminal and run the following command.
+
+     ```js
+     npm ci
+     ```
+
+     This will install the local package dependencies in the Admin applications.
+
+3. Build with customizations
```
Comment (samgnaniah) [inline en/docs/develop/customizations/advanced-ui-customization.md:95]: Let's add what we are building and what we are customizing here.

File: en/docs/develop/customizations/advanced-ui-customization.md [hunk 2]
```diff
@@ -78,6 +78,82 @@ The rest of the files are runtime configurations for eslint, jest, webpack, npm.
...
+You can import the **NewFile.jsx** by adding the **AppOverride** prefix to the import and provide the full path relative to the override directory.
+
+```sh
+import NewFile from 'AppOverride/src/app/components/Apis/Details/NewFile.jsx';
+```
+
+A bundler error will show up if you try to import the **NewFile.jsx** from **Overview.jsx** as follows.
+
+```sh
+import NewFile from './NewFile.jsx';
```
Comment (samgnaniah) [inline en/docs/develop/customizations/advanced-ui-customization.md:155]: Close this code block.

File: en/docs/develop/customizations/advanced-ui-customization.md [hunk 3]
```diff
@@ -78,6 +78,82 @@ The rest of the files are runtime configurations for eslint, jest, webpack, npm.
...
+                    ├── Documents
+                    │   └── Documentation.jsx
+                    └── Overview.jsx
+                    └── NewFile.jsx
+                    
+```
+
+You can import the **NewFile.jsx** by adding the **AppOverride** prefix to the import and provide the full path relative to the override directory.
+
+```sh
```
Comment (samgnaniah) [inline en/docs/develop/customizations/advanced-ui-customization.md:148]: I assume this is Java? Does "sh" render the correct format?

File: en/docs/develop/customizations/advanced-ui-customization.md [hunk 4]
```diff
@@ -78,6 +78,82 @@ The rest of the files are runtime configurations for eslint, jest, webpack, npm.
...
+└── src
+    ├── Readme.txt
+    └── app
+        └── components
+            └── Apis
+                └── Details
+                    ├── Documents
+                    │   └── Documentation.jsx
+                    └── Overview.jsx
+```
```
Comment (samgnaniah) [inline en/docs/develop/customizations/advanced-ui-customization.md:127]: Shall we add some explanation here? This has only a file structure so it's unclear what we're asking them to do when you consider the sub heading.

File: en/docs/develop/customizations/advanced-ui-customization.md [hunk 5]
```diff
@@ -78,6 +78,82 @@ The rest of the files are runtime configurations for eslint, jest, webpack, npm.
...
+    └── app
+        └── components
+            └── Apis
+                └── Details
+                    ├── Documents
+                    │   └── Documentation.jsx
+                    └── Overview.jsx
+                    └── NewFile.jsx
+                    
+```
```
Comment (samgnaniah) [inline en/docs/develop/customizations/advanced-ui-customization.md:144]: Same comment as above. Better if we can add some kind of description.

File: en/docs/develop/customizations/advanced-ui-customization.md (hunk 3 above)
Comment (ruthryi) [reply en/docs/develop/customizations/advanced-ui-customization.md:148]: Yes. This is the correct format.

**Review by samgnaniah — APPROVED** [review]

**Pull request merged**